### ISSUE TITLE
Request info on training data used for pre-trained models

### ISSUE THREAD

**Issue by rppravin** [auspicious3000/autovc#69]
Thanks for the code.

Can you please give info on the data used for training the pre-trained models, both for AutoVC and speaker embedding? If you trained on a subset of a larger database, please let me know such info as well.

Best,
Pravin

**Comment by ruclion** [comment]
In author's Paper, 

- speaker embedding is trained by "the combination of VoxCeleb1 (Nagrani et al., 2017) and Librispeech (Panayotov et al., 2015) corpora, where there are a total of 3549 speakers"
- vocoder: pre-trained the WaveNet vocoder using the method described in Shen et al. (2018) on the VCTK corpus
- autoVC(content encoder&decoder):  VCTK corpus, which has 109 speakers; but in paper, one task uses 20 speakers, another task uses 40speakers; And we don't know which speakers author used

We are trying to re-implement the same loss on VCTK, let's find out more wisdom on author's work togethor~
